proc.c

```diff
@@ -439,13 +439,13 @@ scheduler(void)
         switch (pr->state)
         {
           case RUNNING:
-            pr->rutime++;
+            pr->rutime += (delta_ticks / QUANTA);
```

Comment:
A quanta is currently 5 ticks, unless you set it to poll every quanta rather than every tick (I haven't, we probably should, but I have no idea where that is).